test/acceptance/features/unbindAppToService.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Unbind an application from a service
+
+    As a user of Service Binding Operator
+    I want to unbind an application from a service
+
+    Background:
+        Given Namespace [TEST_NAMESPACE] is used
+        * Service Binding Operator is running
+        * PostgreSQL DB operator is installed
+
+    @disabled
+    Scenario: Unbind a generic test application from PostgreSQL database
+        Given Generic test application "generic-app-a-d-u" is running
+        * DB "db-demo-a-d-u" is running
+        When Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-a-d-u
+            spec:
+                application:
+                    name: generic-app-a-d-u
+                    group: apps
+                    version: v1
+                    resource: deployments
+                services:
+                -   group: postgresql.baiju.dev
+                    version: v1alpha1
+                    kind: Database
+                    name: db-demo-a-d-s
+            """
+        * application should be connected to the DB "binding-request-a-d-u"
```

Comment:
generic test application cannot be connected to db, simply because it does not use it at all. Instead of database, we should an artificial backend, as we do in for example: https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/customEnvVar.feature#L11 This way we can check the existence of some env var inside the application container, or non-existence after the binding got removed.

Reply:
changed logic